documentation/src/main/asciidoc/topics/code_examples/HotRodClientInstance.java

```diff
@@ -1,10 +1,10 @@
-org.infinispan.client.hotrod.configuration.ConfigurationBuilder cb
-      = new org.infinispan.client.hotrod.configuration.ConfigurationBuilder();
-cb.marshaller(new org.infinispan.commons.marshall.ProtoStreamMarshaller())
-  .statistics()
-      .enable()
-      .jmxDomain("org.example")
-  .addServer()
-      .host("127.0.0.1")
-      .port(11222);
-RemoteCacheManager rmc = new RemoteCacheManager(cb.build());
+ConfigurationBuilder builder = new ConfigurationBuilder();
+builder.addServer()
+         .host("127.0.0.1")
+         .port(ConfigurationProperties.DEFAULT_HOTROD_PORT)
```

Comment:
it is missing `192.0.2.0:11222`
the URI example has 2 servers and the `hotrod-client.properties` also has 2 servers.